src/Leela.cpp

```diff
@@ -82,6 +80,9 @@ static void parse_commandline(int argc, char *argv[]) {
         ("logfile,l", po::value<std::string>(), "File to log input/output to.")
         ("quiet,q", "Disable all diagnostic output.")
         ("noponder", "Disable thinking on opponent's time.")
+        ("benchmark", "Test network and exit. Default args:\n-p1600 --noponder "
+                      "-m0 -t1 -s1.\nStop other processes for reproducible "
```

Comment:
Does this imply the GPU is sensitive to context swaps?

Reply:
Not sure, I was thinking more CPU and GPU. If the CPU or GPU is tied up by autogtp or something else, you won't get a good n/s reading. My n/s is always stable when nothing else is running.

Reply:
I would assume that's fairly obvious when benchmarking, was just wondering if something else was implied here.

Reply:
Nope, since you feel it's obvious I've removed the sentence.